.circleci/config.yml

```diff
@@ -51,11 +69,47 @@ jobs:
           destination: test-results
       - store_test_results:
           path: test-results
+  container-validation:
+    machine:
+      image: circleci/classic:latest
+    steps:
+      - checkout
+      - run:
+          name: Setup python3
+          command: |
+            pyenv global 2.7.12 3.5.2
+            python --version
+            pip --version
+            python3 --version
+            pip3 --version
+      - run:
+          name: Setup Tests / Scanner Requirements
+          command: |
+            make test_setup
+      - run:
+          name: Build Container
+          command: |
+            make all
+      - run:
+          name: Export Build Images for Artifacts
+          command: |
+            make save_containers
+      - run:
+          name: Test Python3 installation
+          command: make test_python3_all
+      - run:
+          name: Test Python2 as the default
+          command: make test_python2_all
+      - store_artifacts:
+          path: test-results
+          destination: test-results
 
 workflows:
   version: 2
-  run_tests:
+  build:
     jobs:
-      - docker-splunk-test
+      - security-scanning
+      - debian10-testing
```

Comment:
can we add redhat8 now that we also publish redhat?

Reply:
we can run redhat testing in parallel